api.md

```diff
@@ -840,7 +846,159 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
+[postgres-parallel-agg]:https://www.postgresql.org/docs/current/parallel-plans.html#PARALLEL-AGGREGATION
+[postgres-rls]:https://www.postgresql.org/docs/current/ddl-rowsecurity.html 
+[postgres-security-barrier]:https://www.postgresql.org/docs/current/rules-privileges.html
+
+>:TIP: You can find the [settings for continuous aggregates][timescaledb_information-cagg] and 
+[statistics][timescaledb_information-caggstats] in `timescaledb_information` views.
+
+#### Examples [](cagg-create-examples)
+Create a continuous aggregate view.
+```sql
+CREATE VIEW cagg_view( timec, minl, sumt , sumh )
+WITH ( timescaledb.continuous, timescaledb.refresh_lag = '5 hours', timescaledb.refresh_interval = '1h' )
+AS
+SELECT time_bucket('1day', timec), min(location), sum(temperature),sum(humidity)
+from conditions
+group by time_bucket('1day', timec), location, humidity, temperature;
+```
+---
+## ALTER VIEW (Continuous Aggregate) [](cagg_alter_view)
+`ALTER VIEW` statement can be used to modify the WITH clause  
+[options](#create-view-with) for the continuous aggregate view.
+
+``` sql
+ALTER VIEW <view_name> SET ( timescaledb.option =  <value> )
+```
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+
+#### Examples [](cagg-alter-examples)
+Set the max interval processed by a materializer job (that updates the continuous aggregate) to 1 week.
+```sql
+ALTER VIEW contagg_view SET (timescaledb.max_interval_per_job = '1 week');
+```
+Set the refresh lag to 1 hour, the refresh interval to 30 minutes and the max 
+interval processed by a job to 1 week for the continuous aggregate.
+```sql
+ALTER VIEW contagg_view SET (timescaledb.refresh_lag = '1h', timescaledb.max_interval_per_job='1week', timescaledb.refresh_interval='30m');
+
+```
+>:TIP: Only WITH options can be modified using the ALTER statment. If
+you need to change any other parameters, drop the view and create a new one.
+
+---
+
+## REFRESH MATERIALIZED VIEW (Continuous Aggregate) [](cagg_refresh_view)
+The continuous aggregate view can be manually updated by using `REFRESH MATERIALIZED VIEW` statement. A background materializer job will run immediately and update the
+ continuous aggregate.
+``` sql
+REFRESH MATERIALIZED VIEW <view_name> 
+```
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
 
+#### Examples [](cagg-refresh-examples)
+Update the continuous aggregate view immediately.
+```sql
+REFRESH MATERIALIZED VIEW contagg_view;
+```
+---
+
+>:TIP: Note that max_interval_per_job and refresh_lag parameter settings are used by the materialization job 
+when the REFRESH is run. So the materialization (of the continuous aggregate) does not necessarily include
+all the updates to the hypertable.
+
+
+## DROP VIEW (Continuous Aggregate) [](cagg_cagg_view)
```

Comment:
the anchor link name here doesn't match what's in the command list (`cagg_drop_view`)

Reply:
fixed.